### PR TITLE
Home page

### LINKED ISSUE #72: Add 'Posts Page' Component
User Stories:

- benedekhalaj/terraforming-mars-forum-api#4 
- benedekhalaj/terraforming-mars-forum-api#7 
- benedekhalaj/terraforming-mars-forum-api#8 

Components:

- benedekhalaj/terraforming-mars-forum-api#68 
- benedekhalaj/terraforming-mars-forum-api#69 
- benedekhalaj/terraforming-mars-forum-api#71 

Methods:

- benedekhalaj/terraforming-mars-forum-angular#6

### LINKED ISSUE #150: Refactor League Post Details page

### LINKED ISSUE #152: Add 'Post' interface
- #4

### LINKED ISSUE #153: Add 'icon' component

### PR DESCRIPTION
**Pull request by dudaskobendeguz**
- closes #72 
- closes #150 
- closes #152
- closes #153

### CODE REVIEW

File: angular/src/app/components/icons/icon/icon.component.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<mat-icon matTooltip="{{ tooltip! }}">{{ matIconType }}</mat-icon>
```
Comment (benedekhalaj) [inline angular/src/app/components/icons/icon/icon.component.html]: exclamation mark missing IMO

File: angular/src/app/components/league-posts/icons/icons.component.html [hunk 2]
```diff
@@ -1,13 +1,6 @@
 <div class="icons">
```
Comment (benedekhalaj) [inline angular/src/app/components/league-posts/icons/icons.component.html]: Please rename this file to league-icons

File: angular/src/app/components/league-posts/league-post-detail/league-post-detail.component.html [hunk 3]
```diff
@@ -1,51 +1,46 @@
 <div class="league-post-container">
-  <mat-spinner *ngIf="isLoadingLeaguePost && isLoadingPostDetails"></mat-spinner>
-  <mat-card *ngIf="leaguePost && postDetails">
-    <mat-card-header>
-      <mat-card-title>{{ leaguePost.name }}</mat-card-title>
-      <mat-card-subtitle>{{ leaguePost.gameType }}</mat-card-subtitle>
-      <img mat-card-avatar src="{{ leaguePost.leagueAdmin.imageSource }}"
-           alt="{{ leaguePost.leagueAdmin.username }}'s profile picture">
-    </mat-card-header>
+  <mat-spinner *ngIf="isLoading"></mat-spinner>
```
Comment (benedekhalaj) [inline angular/src/app/components/league-posts/league-post-detail/league-post-detail.component.html:2]: nice loading

File: angular/src/app/components/posts/post-container/post-container.component.ts [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  styleUrls: ['./post-container.component.css']
+})
+export class PostContainerComponent implements OnInit {
+  public posts:Post[] = [];
+  public isLoading: boolean = true;
+  public readonly sortOptions = {
+    'newest': () => {this.filterByDate(true)},
+    'oldest': () => {this.filterByDate(false)},
+    'shuffle': () => {this.shuffle()},
+  };
```
Comment (benedekhalaj) [inline angular/src/app/components/posts/post-container/post-container.component.ts]: This is sort I guess not filter

**Review by benedekhalaj — APPROVED** [review]

**Review by benedekhalaj — CHANGES_REQUESTED** [review]
See previous comments

File: angular/src/app/components/posts/post-container/post-container.component.ts (hunk 4 above)
Comment (dudaskobendeguz) [reply angular/src/app/components/posts/post-container/post-container.component.ts]: Ohhh yep :dango:

File: angular/src/app/components/icons/icon/icon.component.html (hunk 1 above)
Comment (dudaskobendeguz) [reply angular/src/app/components/icons/icon/icon.component.html]: Roger Roger !

**Review by benedekhalaj — COMMENTED** [review]
This is the work I like to see in projects, great example, thank you :dancing_women: :heart:

**Review by benedekhalaj — APPROVED** [review]

**Pull request merged**